rskj-core/src/main/resources/config/main.conf

```diff
@@ -9,6 +9,7 @@ blockchain.config {
         twoToThree = 2018000,
         papyrus200 = 2392700
         iris300 = 3614800
+        hop400 = 0
```

Comment:
until we have a final activation number, we can put `-1` here. Otherwise, it'd be activated starting from block `0`, which is a consensus break